trace/opentracing_test.go

```diff
@@ -236,8 +236,8 @@ func TestTracerInjectExtractHeader(t *testing.T) {
 func TestTraceExtractHeaderEnvoy(t *testing.T) {
 	tracer := Tracer{}
 	tm := textMapReaderWriter(map[string]string{
-		"x-request-id":      "12345",
-		"x-client-trace-id": "67890",
+		"ot-tracer-traceid": "12345",
+		"ot-tracer-spanid":  "67890",
```

Comment:
eg to get decimal IDs of 12345 and 67890, we need these header strings to be 3039 and 10932